docs/migration/2_0.md

```diff
@@ -149,6 +149,7 @@ Support for functions dropped:
 - `CartVoucherService` now requires new parameter `ActiveCartService`. This service needs to be provided for `CartVoucherService`.
 - `CartCouponComponent` no longer uses `CartService`. This service usage was replaced with corresponding methods from `ActiveCartService`. Also `CustomerCouponService` and `FeatureConfigService` are now required parameters. These services needs to be provided for `CartCouponComponent`.
 - `CartItemListComponent` no longer uses `CartService`. This service usage was replaced with corresponding methods from `ActiveCartService.` Also `SelectiveCartService` and `FeatureConfigService` are now required parameters. These services needs to be provided for `CartItemListComponent`.
+- `LogoutGuard` no longer uses `FeatureConfigService`. This service needs to be provided for `LogoutGuard`.
```

Comment:
> This service needs to be provided for `LogoutGuard`.

I'm confused about this, as the `FeatureConfigService` should no longer be used. I'd make it:

"`LogoutGuard` no longer uses `FeatureConfigService` as it was used previously to determine the feature flag."

Reply:
Thanks! will fix as I didn't know what to say with it and went ahead to make it somehow similar to the others